File: front-end/src/service/secaoTIService.js
import axios from "./api";

/** Classe service para as seções de TI */
class SecaoTIService {

    /** Função para buscar todas as seções de TI salvas */
    async getAll() {
        return (await axios.get(`/secao_ti`, { withCredentials: true })).data;
    }
}

export default new SecaoTIService();

File: back-end-maven/src/main/java/net/weg/wegssm/model/service/SecaoTIService.java
package net.weg.wegssm.model.service;

import lombok.AllArgsConstructor;
import net.weg.wegssm.model.entities.SecaoTI;
import net.weg.wegssm.repository.SecaoTIRepository;
import org.springframework.stereotype.Service;

import java.util.List;

/**
 * Classe service para as seções de TI
 */
@Service
@AllArgsConstructor
public class SecaoTIService {

    /**
     * Classe repository das seções de TI
     */
    public SecaoTIRepository secaoTIRepository;

    /**
     * Função para buscar todas as seções de TI
     *
     * @return
     */
    public List<SecaoTI> findAll() {
        return secaoTIRepository.findAll();
    }

}
